README.md

```diff
@@ -69,109 +69,115 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- WindowWalker has more information in the tooltip, additional features like killing the process and closing the window, additional settings and improvements. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Unit converter now accepts additional ways of writing feet and gallon. Also interprets as either imperial or US gallons depending on current culture. Thanks [@yifan-k](https://github.com/yifan-k)!
+- Unit converter now accepts metre for meter.
+- Localization for Web Search and Unit Converter has been enabled.
+- Localization for Windows Terminal has been enabled. (This was a hotfix for 0.56)
+- Calculator now tries to always interpret the dot (.) symbol as a decimal separator, despite configured culture, to meet expectations.
+- Calculator now handles trailing zeroes on hexadecimal numbers correctly.
+- System can now get the local ip and mac addresses. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Folder has improved results, with improved tooltips. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Setting has added entries for Screen Saver and Connect Wiring Display Panel. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Plugins can now show descriptions for their configurations in settings. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Fix for the focus issue when calling PowerToys Run for the first time after login and after returning from some windows.
+- Fix for a bug on Program when creating a shortcut.
+- Validated that upgrading to .net framework 6 fixed the error appearing when shutting down the system with PowerToys Run running.
 
 ### Video conference mute
 
-- nVidia Broadcast software won't crash anymore
+- Newly added microphones are now updated and tracked by VCM.
 
 ### Settings
 
-- Upgrade notes in OOBE
-- Fix for settings being lost (This was a hotfix for 0.55)
-- UX improvements. Thanks [@niels9001](https://github.com/niels9001)
+- What's new button in the bottom with a new look, with a couple more UI tweaks. Thanks [@niels9001](https://github.com/niels9001)!
+- Fixed a bug causing Settings not to open when a racing condition caused Keyboard Manager settings to not be read correctly.
+- To open settings from the tray icon a double click is needed instead of a single click.
+- Fix for a bug which would cause checking for updates to run indefinitely.
+- When auto-updating, pass a flag to avoid restarting without being prompted.
 
 ### Installer
 
-- Believe we have a fix for the long hated "app.dark.png is missing" error. Thanks to [@robmen](https://github.com/robmen) for having a great blog!
-- Installer will launch PowerToys under appropriate elevation versus Admin only due to UAC prompt for installation (This was a hotfix for 0.55)
-- PowerToys will now start if installed under different user
+- Dependencies installers are now called with /norestart to avoid unprompted restarts. (This was a hotfix for 0.56). Thanks [@franky920920](https://github.com/franky920920)!
```

Comment:
```suggestion
- Dependencies installers are now executed with /norestart to avoid unprompted restarts. (This was a hotfix for 0.56). Thanks [@franky920920](https://github.com/franky920920)!
```